### PR TITLE
search but with better data

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

N/A

This mainly just handles the correct search facets as they will be returned from the API... probably will change a little to reduce code... also updates the fake data from a recent locally ran api search response. This also will change slightly as we build it out.

Next step would be to tie in a dynamic api endpoint and have the search work when run with `docker-compose up` but keep using the fake data on stage until a staging api is available

### CODE REVIEW

File: client/src/components/SearchResult.js [hunk 1]
```diff
@@ -180,10 +180,11 @@ export const RequestRequirements = ({ resource }) => {
 
   const requirements = []
   const MTA = 'Material Transfer Agreement'
-  if (resource.needs_abstract) requirements.push('Abstract')
-  if (resource.needs_irb) requirements.push('IRB')
-  if (resource.needs_mta) requirements.push(MTA)
-  if (resource.needs_shipping_info) requirements.push('Shipping Information')
+  // if (resource.needs_abstract) requirements.push('Abstract')
+  // if (resource.needs_irb) requirements.push('IRB')
```
Comment (kurtwheeler) [inline client/src/components/SearchResult.js:184]: Any particular reason for these to hang around?

Comment (davidsmejia) [reply client/src/components/SearchResult.js:184]: I didn't see them in the response yet I didn't know if I am waiting for an attachment for them or a different attribute name

Comment (kurtwheeler) [reply client/src/components/SearchResult.js:184]: Oh, looks like they're just missing from the elasticsearch view. I just push an UNTESTED :scream: commit to address that. Can you test it? (I'm working on another branch with migrations right now.)

Comment (davidsmejia) [reply client/src/components/SearchResult.js:184]: yeah, ill update the testData too

**Comment by kurtwheeler** [comment]
Why are you doing this instead of just hitting the search endpoint?

**Pull request merged**